enterprise/index.md

```diff
@@ -4,17 +4,137 @@ description: Information about Docker Enterprise Edition
+to run more technology in containers that are supported both by Docker and
+publishers.
+
+[Docker Store](https://store.docker.com/) provides a marketplace for publishers to distribute, sell, and manage their listings, and it enables customers to browse, evaluate, and purchase publisher technologies as containers.
+
+Certified containers and plugins have visible badges, so customers can
+quickly find certified content. Customers can be confident that badged
+content is built with best practices and tested to operate smoothly on Docker
+EE. Customers manage all subscriptions, including those for Docker products
+and publisher content, in a single place.
+
+Three categories of Docker Certified technology are available:
+
+1.  **Certified Infrastructure:** Includes cloud providers and operating
+    systems that the Docker platform is integrated with, optimized for, and
+    tested for certification. With certified infrastructure, Docker provides
+    a great user experience and preserves application portability.
+2.  **Certified Containers:** ISVs can package and distribute their software
+    as containers directly to the end user. These containers are tested, built
+    with Docker-recommended best practices, scanned for vulnerabilities, and
+    reviewed before posting on Docker Store.
+3.  **Certified Plugins:** Networking and Volume plugins for Docker EE are
+    available to be packaged and distributed to end users as containers. These
+    plugin containers are built with Docker-recommended best practices,
+    scanned for vulnerabilities, and must pass an additional suite of API
+    compliance testing before they're reviewed and posted on Docker Store.
+    Apps are portable across different network and storage infrastructure, and
+    they work with new plugins without recoding.
+
+The Docker Certification Program broadens the Docker platform to a wide
+community of developers and IT ops teams at businesses of all sizes.
+Businesses that seek to modernize their applications can build and deploy on
+Docker EE for Linux and Windows, on any infrastructure. Many components of
+enterprise environments come from third parties, and the Docker Certified
+Program accelerates the adoption of these technologies into Docker
+environments with assurances and support.
+
+## Docker EE feature tiers
+
+Docker EE is available in three tiers:
+
+-  **Basic:** The Docker platform for certified infrastructure, with support
+   from Docker Inc. and certified containers and plugins from Docker Store.
+-  **Standard:** Adds advanced image and container management, LDAP/AD user
+   integration, and role-based access control. Together, these features
+   comprise *Docker Datacenter*.
+-  **Advanced:** Adds
+   [Docker Security Scanning](https://blog.docker.com/2016/05/docker-security-scanning/)
+   and continuous vulnerability monitoring.
+
+## Docker Enterprise Edition and Docker Community Edition release cycles
```

Comment:
I think it's valuable to document the EE release cycle, but I'm unsure if it's worth also having the CE stuff. @mstanleyjones is documenting that separately, I think